tests/unit/CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ if(HIGHFIVE_PARALLEL_HDF5)
   target_link_libraries(tests_parallel_bin
     ${Boost_UNIT_TEST_FRAMEWORK_LIBRARIES} ${HDF5_C_LIBRARIES}
     ${MPI_C_LIBRARIES})
-  add_test(NAME tests_parallel COMMAND ${TEST_MPI_EXEC_PREFIX_DEFAULT} -n 8
+  add_test(NAME tests_parallel COMMAND ${TEST_MPI_EXEC_PREFIX_DEFAULT}
```

Comment:
without `-n 8`, is the test still executed in // ?

Reply:
srun and mpiexec take by default the size of the allocation. If you prefer to hardcode the value then I'd suggest 2, since 8 cores is still not very common

Reply:
I am good